### ISSUE TITLE
Consider using max possible message size for TFramedTransports

### ISSUE THREAD

**Issue by ctubbsii** [apache/accumulo#3791]
> Since we're only using the framed transport because we have to, I'm wondering if we should just always set the max message size to the maximum value possible. If we're only using this transport because we have to for the server type, and not because actually want to make use of its features, like size limitations, then it'd probably be easier if we just removed the configuration option, and always set the max size possible. At the very least, we could make it so the default value is always the max size possible, without removing the config. I think one of these options would make things a lot easier on users who are hitting this.

_Originally posted by @ctubbsii in https://github.com/apache/accumulo/issues/3762#issuecomment-1739951104_

**Comment by EdColeman** [comment]
@ctubbsii - should we make this a blocker for 2.1.3?

**Comment by dlmarion** [comment]
FWIW - As this is a configurable property, and #3873 just changes the default value from `1G` to `Integer.toString(Integer.MAX_VALUE)`, this may not need to be a blocker for 2.1.3. I think it's more of a nice to have when it's ready.

**Comment by ctubbsii** [comment]
I have a completed fix for this and all the related issues, but it depends on a related fix for how isPropertySet works (or rather, doesn't work) for fixed properties, which applies to these max message properties, and affects how deprecating the properties needs to be handled. That related fix is in #4752 and needs to be merged first.
